samples/14.nlp-with-dispatch/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+MicrosoftAppId=
+MicrosoftAppPassword=
+server.port=3978
```

Comment:
Can we move this at the last property here?

Reply:
Sure, I'll do it